views/includes/page-templates/beneficial-owner.html

```diff
@@ -38,11 +38,13 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
       <button class="govuk-button" data-module="govuk-button" id="add" formaction="{{ addButtonAction }}" >Add<span class="govuk-visually-hidden"> {{ whatToAdd }} </span></button>
 
       {% if (hasBeneficialOwners or hasManagingOfficers) %}
-        <h2 class="govuk-heading-l govuk-!-margin-0">What <span class="govuk-visually-hidden"> {{ hiddenTextValue }}</span> you have added so far:</h2>
-        <br>
-        {% include "includes/list/beneficial-owner.html" %}
-        {% include "includes/list/managing-officer.html" %}
-
+        
+        {% if not noLists %}
```

Comment:
This is not needed. 

Adding "no more to add button" outside if block on line 45 is sufficient"

Reply:
We'll be working on expanding this page with the review BO/MO logic so it's wip.